raspberry-pi/5/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, pkgs, ... }:
+
+{
+  nixpkgs.overlays = [
+    (import ./overlay.nix)
+  ];
+
+  boot = {
+    kernelPackages = lib.mkDefault (pkgs.linuxPackagesFor pkgs.linux_rpi5);
```

Comment:
We should probably add an assertion like https://github.com/NixOS/nixos-hardware/blob/68d680c1b7c0e67a9b2144d6776583ee83664ef4/raspberry-pi/4/default.nix#L43 that a minimum Linux kernel version ( i only saw 6.1.63 work here

Reply:
Done. Nixpkgs skipped from 6.1.21 to 6.1.63, but I used 6.1.54 which appears to be the first raspberrypi/linux version with support.